### ISSUE TITLE
[bitnami/nginx-ingress-controller] Access-Logs are gone with switch from 1.3.0 to 1.4.0

### ISSUE THREAD

**Issue by aeimer** [bitnami/charts#12989]
### Name and Version

nginx-ingress-controller 9.3.18

### What steps will reproduce the bug?

Deploy nginx controller and check for access logs.

### Are you using any custom parameters or values?

```yaml
---
service:
  # Makes sure that we get the client IP at the ingress-controller-pod
  externalTrafficPolicy: Local

defaultBackendService: foo/bar-default-be
defaultBackend:
  enabled: false

ingressClassResource:
  name: nginx-main
  controllerClass: k8s.io/ingress-nginx-main

config:
  # Add these static headers always
  add-headers: ingress/add-custom-headers

  # Increase client_body_buffer_size to get rid of warnings
  # => By default, buffer size is equal to two memory pages.
  client-body-buffer-size: 32k  # default: 16K on 64bit machines

  # Enable better alternative to gzip
  enable-brotli: 'true'

  # Enable OCSP stapling
  enable-ocsp: 'true'

  # Add X-Request-ID HTTP Header to proxy target call, if none existing
  generate-request-id: 'true'

  limit-req-status-code: '429'  # default 503

  # Use injected secret for Diffie-Hellman
  ssl-dh-param: ingress/ingress-diffie-hellmann

  # pod startup flag needed
  use-geoip2: 'true'

  # Use gzip, may also activate brotli when fixed
  use-gzip: 'true'

  # nginx Logformat in logfmt
  log-format-upstream: >-
    ts="$time_iso8601"
    type=upstream
    stat=$status
    host="$host"
    req="$request"
    r_addr="$remote_addr"
    r_cntry=$geoip2_city_country_code
    r_usr="$remote_user"
    bb_sent=$body_bytes_sent
    ref="$http_referer"
    ua="$http_user_agent"
    fwd_for="$http_x_forwarded_for"
    upstr_addr="$upstream_addr"
    upstr_stat="$upstream_status"
    upstr_res_t="$upstream_response_time"
    upstr_res_len="$upstream_response_length"
    upstr_name="$proxy_upstream_name"
    upstr_name_alt="$proxy_alternative_upstream_name"
    req_t="$request_time"
    req_len="$request_length"
    i_trc_id=$http_x_instana_t

  log-format-stream: >-
    ts="$time_iso8601"
    type=stream
    r_addr="$remote_addr"
    proto=$protocol
    b_sent="$bytes_sent"
    b_rcvd="$bytes_received"
    session_t="$session_time"

  # We are blocking:
  # ---
  server-snippet: |
    if ($geoip2_city_country_code ~* "(FOO)|(BAR)"){
      set $blocked 1;
    }
    if ($blocked = 1) {
      return 403;
    }

metrics:
  enabled: true
  serviceMonitor:
    enabled: true

```

### What is the expected behavior?

The access-logs should be printed (in the defined format) as beforehand.

I already checked, renaming/removing the logstyle properties in the CM does not help. Also I dont see an error logline in the logs.

### What do you see instead?

I only see the go related outputs, like which ingress and secret was loaded.

**Comment by javsalgar** [comment]
Hi,

Just to confirm, with the exact configuration but just changing the image to 1.3.0, the issue does not happen, right?

**Comment by balazs92117** [comment]
I can confirm that with chart version 9.2.28 (app version 1.3.0) it does work!

**Comment by javsalgar** [comment]
Hi,

Could you try with the latest version of the chart but changing the `image` parameter? I think it has to do with the container image and not the chart itself but I'd like to confirm.

**Comment by balazs92117** [comment]
@javsalgar I can confirm that with latest chart (9.3.18) and old image (1.3.0-debian-11-r9) the access log works!

**Comment by ranjith-suram** [comment]
tac-bitnami/nginx-ingress-controller:1.4.X no longer has error and access log symlink to /dev/stderr & /dev stdout

![IngressController1 4-vs-1 3](https://user-images.githubusercontent.com/534458/197585016-272d5117-a1ae-4bf7-aada-8ad1753b8cb6.png)


As a workaround you may apply the nginx Kubectl config 

  access-log-path: "/dev/stdout"
  error-log-path: "/dev/stderr"

to update nginx.conf to point to stdout & stderr

**Comment by gongomgra** [comment]
Hi @aeimer, @balazs92117, @ranjith-suram,

We released a new container linking the logs to stdout/stderr as it was done before. We performed some changes in the image for the 1.4.0 release and it looks like we removed the symlinks by mistake. Starting from  the`1.5.1-debian-11-r5` tag, all our images should include the change.

Also find below the link to the exact commit updating the Dockerfile.

https://github.com/bitnami/containers/commit/4712cee0176c9a3dc106a2ac98e35996ba52fe06
